src/components/ImportGithub.js

```diff
@@ -0,0 +1,60 @@
+import { Fragment, useState } from 'react'
+import { div, h, h2 } from 'react-hyperscript-helpers'
+import { ButtonPrimary } from 'src/components/common'
+import { TextInput } from 'src/components/input'
+import { submitMethod } from 'src/components/method-common'
+import { withBusyState } from 'src/libs/utils'
+
+// { tooltip: 'Link must start with https://github.com or https://raw.githubusercontent.com' },
+const ImportGithub = ({ setLoading, signal, onDismiss }) => {
+  const [methodName, setMethodName] = useState('')
+  const [methodVersionName, setMethodVersionName] = useState('')
+  const [methodUrl, setMethodUrl] = useState('')
+
+  return div({ style: { marginLeft: '4rem' } }, [
+    div({ style: { width: 500 } }, [h2(['Workflow Link'])]),
+    div({}, [
+      h(TextInput, {
+        style: { width: 500 },
+        placeholder: 'Paste Github Link',
+        value: methodUrl,
+        onChange: u => setMethodUrl(u),
+        'aria-label': 'Github link input'
+      })
+    ]),
+    div({ style: { marginTop: '3rem', width: 500 } }, [
+      h2(['New Workflow Name / Version'])
```

Comment:
small nit: I feel the `h2` is making the input names appear too big and out of place in context to the sub-headers `Browse Suggested Workflows` and `Add a Workflow link` and the text size in Helpful Hints box. Maybe `h3` would be a better heading format here? What do you think?
h2:
![Screenshot 2023-05-05 at 11 10 17 AM](https://user-images.githubusercontent.com/16748522/236497212-80624624-c35a-4098-8310-f27bb14e8744.png)
h3:
![Screenshot 2023-05-05 at 11 10 58 AM](https://user-images.githubusercontent.com/16748522/236497408-04bc140f-6500-423f-a217-beacfc0bf3ac.png)